server/routes/eligibility-checker/cannot-trade.route.js

```diff
@@ -32,15 +33,21 @@ const handlers = {
 }
 
 const _getContext = async request => {
-  const species = (await RedisHelper.getSpecies(request)).toLowerCase()
   const referringUrl = request.headers.referer
 
   const pageTitle = 'You are not allowed to sell or hire out your item'
 
   if (referringUrl.includes(Paths.TAKEN_FROM_SPECIES)) {
+    const speciesValue = (await RedisHelper.getSpecies(request)).toLowerCase()
+    let speciesString = 'species'
+
+    if (Object.values(Species).map(item => item.toLowerCase()).includes(speciesValue)) {
+      speciesString = speciesValue
+    }
```

Comment:
This could be a candidate for the rule of three?

https://github.com/DEFRA/software-development-standards/blob/master/standards/common_coding_standards.md#all-code-is-checked-for-simplicity

Reply:
Sure. I'll break this out into a smaller reusable function